applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -290,7 +286,7 @@ static int lc3_config_cb(struct bt_conn *conn, const struct bt_audio_ep *ep, enu
 			else if (dir == BT_AUDIO_DIR_SOURCE) {
 				LOG_DBG("BT_AUDIO_DIR_SOURCE");
 				print_codec(codec);
-				sources[configured_source_stream_count++].stream = audio_stream;
+				sources[0].stream = audio_stream;
```

Comment:
This may be fine, but I would appreciate a short explanation.

Reply:
In the very beginning, we increased the index once if a stream is configured. However, the clean-up doesn't work properly so there's a refactor on this part in here: https://github.com/nrfconnect/sdk-nrf/pull/9178
So we use `audio_stream_index_get() ` for getting the index to distinguish the sink stream for left or right.
But for source stream, since there's only one stream now, so it will be 0 at this moment.

Reply:
In the very beginning, we increased the index once if a stream is configured. However, the clean-up doesn't work properly so there's a refactor on this part in here: https://github.com/nrfconnect/sdk-nrf/pull/9178
So we use `audio_stream_index_get() ` for getting the index to distinguish the sink stream for left or right.
But for headset side, this might missed, also for source stream, since there's only one stream now, so it will be 0 at this moment.

Reply:
We should do a BUILD_ASSERT() for CONFIG_BT_ASCS_ASE_SRC_COUNT > 1

See cis_gateway code for reference:
https://github.com/nrfconnect/sdk-nrf/blob/main/applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c#L58